impl/meson.build

```diff
@@ -267,7 +269,7 @@ if meson.is_cross_build()
 		gnu_symbol_visibility: 'inlineshidden',
 		implicit_include_directories: false,
 		pic: true,
-		install: (not meson.is_subproject()),
+		install: false,
```

Comment:
Was this intentional? I can foresee this causing installation problems where the library is not installed despite the headers that need it being there.

Reply:
Yes, see the commit message for cc303f4 - with the code it's replacing, Meson will refuse to complete configuration. Note, it is for the native (build host) build during cross-compilation rather than the host build version.

Reply:
Ah, okay, lgtm then